packages/labs/testing/src/lib/fixtures/ssr-fixture.ts

```diff
@@ -54,7 +54,9 @@ export async function ssrFixture<T extends HTMLElement>(
     // @http://localhost:8000/test/my-element_test.js?wtr-session-id=aKWON-wBOBGyzb2CwIvmK:65:37
     const {stack} = new Error();
     const match =
-      stack?.match(/http:\/\/localhost.+(?=\?wtr-session-id)/) ??
+      stack?.match(
+        /http:\/\/(localhost|host.containers.internal).+(?=\?wtr-session-id)/
```

Comment:
should the `.` be escaped? i don't have a quick way of testing this. it probably works without it since it's any char, but seems more proper.
```suggestion
        /http:\/\/(localhost|host\.containers\.internal).+(?=\?wtr-session-id)/
```

Reply:
Nice catch. Thanks!